dockerfile-generator/dockerfile-gen-services/src/main/java/io/hyscale/dockerfile/gen/services/exception/DockerfileErrorCodes.java

```diff
@@ -15,28 +15,40 @@
  */
 package io.hyscale.dockerfile.gen.services.exception;
 
-import io.hyscale.commons.exception.HyscaleErrorCode;
+import io.hyscale.commons.exception.HyscaleError;
+import io.hyscale.commons.exception.HyscaleErrorGroup;
 
-public enum DockerfileErrorCodes implements HyscaleErrorCode {
-    FAILED_TO_GENERATE_DOCKERFILE("Failed to generate Dockerfile"),
+public enum DockerfileErrorCodes implements HyscaleError {
+    FAILED_TO_GENERATE_DOCKERFILE("Failed to generate Dockerfile",HyscaleErrorGroup.DOCKER_FILE_GENERATION),
     FAILED_TO_COPY_ARTIFACT("Failed to copy artifacts"),
     FAILED_TO_CREATE_SCRIPT("Failed to create script"),
     FAILED_TO_PERSIST_DOCKERFILE("Failed to persist Dockerfile"),
```

Comment:
These all failures come under the same group DOCKER_FILE_GENERATION

Reply:
As per our initial discussion, I did not assigned the error groups for those being not used anywhere. So I ignored the above.  Please let me know whether to assign those or not?